frontend-react/src/components/Crumbs.tsx

```diff
@@ -11,28 +16,49 @@ interface CrumbConfig {
 interface CrumbsProps {
     crumbList?: CrumbConfig[];
     noPadding?: boolean;
+    previousPage?: string;
 }
 
-const Crumbs = ({ crumbList, noPadding }: CrumbsProps) => {
-    if (crumbList) {
+const Crumbs = ({ crumbList, noPadding, previousPage }: CrumbsProps) => {
+    if (crumbList || previousPage) {
         return (
             <div className={!noPadding ? "grid-container" : ""}>
-                <BreadcrumbBar>
-                    {crumbList?.map((crumbConfig) => (
-                        <Breadcrumb
-                            key={`${crumbConfig.label}`}
-                            current={!crumbConfig?.path}
+                {Boolean(previousPage) ? (
+                    <div className="font-sans-lg line-height-sans-4 margin-top-8">
+                        <IconButton
+                            size="big"
+                            type="button"
+                            onClick={() => window.history.back()}
                         >
-                            {crumbConfig?.path ? (
-                                <USCrumbLink href={crumbConfig?.path || ""}>
-                                    {crumbConfig.label}
-                                </USCrumbLink>
-                            ) : (
-                                crumbConfig.label
-                            )}
-                        </Breadcrumb>
-                    ))}
-                </BreadcrumbBar>
+                            <IconArrowBack />
+                        </IconButton>
+                        Return to {previousPage}
+                    </div>
+                ) : null}
+                {crumbList !== undefined && crumbList.length > 0 ? (
+                    <>
+                        <hr />
+                        <BreadcrumbBar>
```

Comment:
Really small thing, but there's a bit of vertical asymmetry on less-than-desktop viewports since `usa-breadcrumb` has bottom padding of 1rem 🤔

<img width="490" alt="Screenshot 2023-01-10 at 13 25 49" src="https://user-images.githubusercontent.com/2421042/211632493-a1c31dbc-941a-406c-ae8f-32b60668a261.png">

Not a blocker in my mind, though!